Tests/run_tests.py

```diff
@@ -165,6 +166,12 @@ def is_numpy():
     DOCTEST_MODULES.remove("Bio.SeqIO")
     DOCTEST_MODULES.remove("Bio.SearchIO")
 
+# Skip doctests for all modules outside Bio & BioSQL prior to Python 3.3
+if float(sys.version[:3]) < 3.3:
```

Comment:
Sorry, I should have spotted this earlier - you should not assume you can break up the version string like that.

Instead use ``sys.version_info`` which is a (named) tuple, i.e. something like this should work?

```python
if sys.version_info < (3, 3):
```